### PR TITLE
NO-ISSUE disk unit test refactor

### PR DESCRIPTION
**Pull request by michaellevy101**
increase code readability

### CODE REVIEW

**Comment by omertuc** [comment]
> /assign otuchfel

/assign @omertuc

File: src/inventory/disks_test.go [hunk 1]
```diff
@@ -43,69 +44,185 @@ func deleteExpectedCall(mockDeps *MockIDependencies, methodName string, argument
...
+		prepareDiskObjects(dependencies, i)
+		blockInfo.Disks = append(blockInfo.Disks, createFakeGHWDisk(i))
+		expectedDisks = append(expectedDisks, createFakeModelDisk(i))
 	}
 
 	return blockInfo, expectedDisks
 }
 
-var _ = Describe("Disks test", func() {
+var _ = FDescribe("Disks test", func() {
```
Comment (omertuc) [inline src/inventory/disks_test.go:225]: Describe

Comment (michaellevy101) [reply src/inventory/disks_test.go]: already done:)

File: src/inventory/disks_test.go [hunk 2]
```diff
@@ -43,63 +44,179 @@ func deleteExpectedCall(mockDeps *MockIDependencies, methodName string, argument
...
+	} else {
+		fileInfoMock := FileInfoMock{}
+		fileInfoMock.On("Name").Return(path).Once()
+		var info os.FileInfo = &fileInfoMock
+		return dependencies.On("Stat", path).Return(info, nil).Once()
+	}
+}
+
+/**
+Mock the dependency call that try to locate the disk at /dev/diskName
```
Comment (omertuc) [inline src/inventory/disks_test.go:154]: Please follow standard go documentation conventions 

https://pkg.go.dev/github.com/natefinch/godocgo
https://blog.golang.org/godoc

**Pull request merged**